prevent compile error:

In file included from /ws/grzemba/oi-userland/components/web/firefox/firefox-91.0.1/ipc/chromium/src/base/message_pump_libevent.cc:20:0,
                 from Unified_cpp_ipc_chromium0.cpp:83:
/ws/grzemba/oi-userland/components/web/firefox/build/amd64/dist/system_wrappers/event.h:3:15: fatal error: event.h: No such file or directory
 #include_next <event.h>
 
diff --git a/ipc/chromium/moz.build b/ipc/chromium/moz.build
index a401245..50bfe72 100644
--- a/ipc/chromium/moz.build
+++ b/ipc/chromium/moz.build
@@ -125,6 +125,7 @@ if os_solaris:
     ]
 
 CXXFLAGS += CONFIG["TK_CFLAGS"]
+CXXFLAGS += CONFIG['MOZ_LIBEVENT_CFLAGS']
 
 TEST_DIRS += [
     "gtest",
